assets/tutorial/01-blockly.js

```diff
@@ -0,0 +1,62 @@
+const entities = require('../../models/game/entities')
```

Comment:
For me the levels don't really feel like assets, because you can execute arbitraryJavaScript here. You also have dependencies on other modules. That's why I originally put them into the tutorial module.